community/index/src/main/java/org/neo4j/index/internal/gbptree/GBPTree.java

```diff
@@ -734,31 +734,15 @@ private void checkpoint( IOLimiter ioLimiter, Header.Writer headerWriter ) throw
     }
 
     /**
-     * Closes this tree and its associated resources. A {@link #checkpoint(IOLimiter)} is first performed
-     * as part of this call if there have been changes since last call to {@link #checkpoint(IOLimiter)}
-     * or since opening this tree.
+     * Closes this tree and its associated resources.
+     * NOTE: No {@link #checkpoint(IOLimiter) checkpoint} is performed. To make data persistent in store
+     * a checkpoint needs to be performed manually.
```

Comment:
Why do you need to mention at all that `To make data persistent in store a checkpoint needs to be performed manually.` if javadoc already say that close does not perform it that sentence does not bring any additional value.